.github/workflows/release-helm.yml

```diff
@@ -0,0 +1,18 @@
+name: Release Helm Chart
+
+on:
+  workflow_dispatch:
+
+jobs:
+  release:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - uses: stefanprodan/helm-gh-pages@v1.7.0
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          charts_dir: "docker/kubernetes"
+          branch: charts
```

Comment:
Does the branch has to be called `charts`? If the name could be chosen I'd go for `helm`.

Reply:
yes that doesn't matter, most use charts